### ISSUE TITLE
Feature Request: ("live") global label (set) management

### ISSUE THREAD

**Issue by mscdex** [go-gitea/gitea#13958]
- Gitea version (or commit ref): 1.13.0
- Git version: n/a
- Operating system: Linux
  
  
  
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description

I'd like to see organization labels-like functionality at the global level. The use case for this is we have several users in a single gitea instance, with each having their own different repositories, some of which share certain organizational attributes. For example, multiple users may have repositories that contain customer-specific data (possibly along with generic data in the same repo). Being able to share "live" customer labels across these repositories would make management so much easier than having to remember to go to each affected repository and add/edit/delete one or more labels when that time comes, all while keeping the repository ownership as-is.

Ideally this feature would include:

* Making all label sets "live", like an organization's label set
* A proper web UI for managing label sets, probably via somewhere in the site admin settings (instead of having to manage files on disk in `custom/`)
* For each label set, a web UI identical to that of organization labels: ability to add/edit/delete any labels, see how many issues/PRs are using each label with a link to those issues/PRs, ability to sort labels, etc.
* Global labels (from global label sets) would of course be visible to all repositories and like organization labels, would not be mutable in each repository

**Comment by viceice** [comment]
Yeah a system label set would help us too. We have an org for each customer and we likle to share some general labels like issue type / state / prio ...
